src/main/java/com/mojang/brigadier/builder/LiteralArgumentBuilder.java

```diff
@@ -9,6 +9,8 @@
 public class LiteralArgumentBuilder<S> extends ArgumentBuilder<S, LiteralArgumentBuilder<S>> {
     private final String literal;
 
+    private boolean isCaseInsensitive = false;
```

Comment:
This should be final. Also rename the variable to `ignoreCase` as that is the convention.